### PR TITLE
Vanguarde layout & Search filters card

### PR DESCRIPTION
**Pull request by xelav**
This PR will make a bunch of small changes to make more sane elements positioning and more consistent design looks.
How it looks now:
image

Though this is out of scope of the given task, I also made a small card with search filters, which is toggled by the button next to search input.

### CODE REVIEW

**Comment by artem-bondar** [comment]
By the way, I'm too lazy to insist that 399adbc and 9321f37 can be squashed.

**Comment by xelav** [comment]
Actually commits having 399adbc and 9321f37 the same message is my mistake. They are different and 9321f37 requires different name.

File: vanguarde/src/components/Footer.jsx [hunk 1]
```diff
@@ -3,13 +3,49 @@ import {MDBContainer, MDBFooter} from "mdbreact";
 
 export class Footer extends Component {
 
+    constructor(props){
+        super(props);
+
+
+        this.footerClicks = 0;
+        this.footerArray = [
+            '© ' + new Date().getFullYear() + ' Copyright LateInit Apps',
```
Comment (artem-bondar) [inline vanguarde/src/components/Footer.jsx:12]: Lateinit, not LateInit, desu.

File: vanguarde/src/components/SongCard.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                                <div className="col-md-6">
+                                    <div className="card-body">
+                                        <h4 className="card-title">{this.props.artist} -  {this.props.trackName} </h4>
+                                    </div>
+                                </div>
+                            </div>
+                        </MDBCard>
+                    </MDBCol>
+                    <MDBCol md="2">
+                        <MDBBtn className={"alignRigh"} color="elegant">
```
Comment (kostmetallist) [inline vanguarde/src/components/SongCard.jsx:31]: Does "alignRigh" need to be renamed as "alignRight"? Also, it is not clear for me why some of class names are presented in curly braces while others are just strings with (as I think) Bootstrap keywords?

**Comment by bozzyk** [comment]
Just a reminder: don't forget to squash when you are merging

**Comment by artem-bondar** [comment]
> Just a reminder: don't forget to squash when you are merging

NO SQUASHING. Simple merges. Preserve all commits, amen.

Nevertherless, that was discussed on meetings and is mentioned in our conventions [here](https://github.com/lateinit-apps/guitar-app/blob/develop/docs/conventions.md#pull-requests-conventions).

File: vanguarde/src/components/SongCard.jsx (hunk 2 above)
Comment (xelav) [reply vanguarde/src/components/SongCard.jsx:31]: Well spotted, this is an insidious typo.

The thing with curly braces, it evaluates JS-code inside, just like python's `format`. In most cases (like this one) it's <s>muda muda</s> useless and probably should be refactored.

File: vanguarde/src/components/Footer.jsx (hunk 1 above)
Comment (xelav) [reply vanguarde/src/components/Footer.jsx:12]: Gotcha

**Pull request merged**